test/spec/adapters/sharethrough_spec.js

```diff
@@ -117,8 +108,8 @@ describe('sharethrough adapter', () => {
                               "stxUserId": ""
                             };
 
-      pbjs.strcallback(JSON.stringify(bidderReponse1));
-      pbjs.strcallback(JSON.stringify(bidderReponse2));
+      adapter.callback(bidderRequest.bids[0], JSON.stringify(bidderReponse1));
```

Comment:
rather than exposing the callback just for testing, it's preferred that you test using the `sinon.fakeServer` with a mock response and calling `adapter.callBids()`.  You can see an example in [here](https://github.com/prebid/Prebid.js/blob/master/test/spec/adapters/rubicon_spec.js#L466).